src/carplay.cs

```diff
@@ -378,8 +382,14 @@ interface CPListTemplateDelegate {
 	[BaseType (typeof (NSObject))]
 	interface CPManeuver : NSCopying, NSSecureCoding {
 
+		[Introduced (PlatformName.iOS, 12,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'CPManeuver.symbolImage' instead.")]
```

Comment:
Just a small casing issue: 

```suggestion
		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'CPManeuver.SymbolImage' instead.")]
```

Reply:
So this deprecated attribute, when we use it and add a message, the user will receive the message when they try to use SymbolSet in iOS13?

Reply:
@tj-devel709 yes, Visual Studio will show this warning depending on the minimum version registered in the app's info.plist